neuraxle/data_container.py

```diff
@@ -0,0 +1,242 @@
+"""
+Neuraxle's DataContainer classes
+====================================
+Classes for containing the data that flows throught the pipeline steps.
```

Comment:
I recommend a new line below the title and the paragraph to avoid possible documentation compilation issue (?)

Reply:
all of our docstrings are already like I did here ? I don't think there is a problem here. resolving